doc/source/guide/data_types/maps.rst

```diff
@@ -137,28 +151,330 @@ more complicated figure.
+data in an image. For example large data value could be due to
+cosmic ray hits and should be ignored. The most straightforward way to ignore
+this kind of data in plots without altering the data is to clip it. This can be achieved
+very easily when initializing the normalization variable. For example::
+
+    norm = colors.Normalize(vmin=smap.min(), vmax=smap.mean() + 3 *smap.std())
+
+This clips out many of the brightest pixels. If you'd like to see what areas of
+your images got clipped set the following values::
+
+    cmap = cmap.plot_settings['cmap']
+    cmap.set_over('red', 1.0)
+    cmap.set_under('green', 1.0)
+
+This will color the areas above and below in red and green respectively
+(similar to this `example <http://matplotlib.org/examples/pylab_examples/image_masked.html>`_).
+You can use the following colorbar command to display these choices::
+
+    plt.colorbar(extend='both')
+
+Here is an example of this put to use on an AIA image. If you see how the image
+displays by default you'll see that it does not look that pretty. This is
+because the image contains some negative values which are throwing off the
+normalization.
+
+.. plot::
+
+    import sunpy.map
+    import matplotlib.pyplot as plt
+    import sunpy.data.sample
+    smap = sunpy.map.Map(sunpy.data.sample.AIA_94_CUTOUT)
+    txt = "min={min}, max={max}, $\mu$={mean}, $\sigma$={std}".format(min=int(smap.min()),
+                                                                      max=int(smap.max()),
+                                                                      mean=int(smap.mean()),
+                                                                      std=int(smap.std()))
+    plt.text(-1100, 0, txt, color='white')
+    smap.plot()
+    plt.colorbar()
+    plt.show()
+
+In order to fix this we need to adjust our normalization to not display negative
+values. We can also brighten the image by clipping the high values though this
+will mean that the bright regions look 'saturated'. This is achieved in the following plot.
+
+.. plot::
+
+    import sunpy.map
+    import matplotlib.pyplot as plt
+    import matplotlib.colors as colors
+    import sunpy.data.sample
+    smap = sunpy.map.Map(sunpy.data.sample.AIA_94_CUTOUT)
+    cmap = smap.plot_settings['cmap']
+    cmap.set_over('blue', 1.0)
+    cmap.set_under('purple', 1.0)
+    norm = colors.Normalize(vmin=0, vmax=smap.mean() + 5 * smap.std())
+    smap.plot(norm=norm)
+    plt.colorbar(extend='both')
+    plt.show()
+
+Another method to ignore bad data is to mask the data. A mask is a boolean
+array and so can give you much more fine-grained control over what is not being
+displayed.  A `~numpy.ma.MaskedArray`
+is a subclass of a numpy array so it has all of the same properties with the
+addition of an associated boolean array which holds the mask.
+
+
+.. The following plot achieves the same goal as above but using a mask instead of clipping.
```

Comment:
Don't need it but wanted to keep it in to add it back in later on. It's a good example idea.